static/docs/get-started/agenda.md

```diff
@@ -1,36 +1,35 @@
 # Agenda
 
-In the next few sections we will build a simple natural language processing
-(NLP) project from scratch. If you'd like to get the final result or have some
-issues along the way, you can download the fully reproducible
-[GitHub project](https://github.com/iterative/example-get-started) with:
+In the next few sections we'll build a simple natural language processing (NLP)
+project from scratch. If you'd like to get the final result or have any issues
+along the way, you can download the fully reproducible
+[GitHub project](https://github.com/iterative/example-get-started) by running:
 
 ```dvc
 $ git clone https://github.com/iterative/example-get-started
 ```
 
-Otherwise, bear with us and we will introduce the basic DVC concepts and get to
+Otherwise, bear with us and we'll introduce some basic DVC concepts to get to
 the same result together!
 
-The idea of the project is a simplified version of the
-[Tutorial](/doc/tutorials/deep). It explores the NLP problem of predicting tags
-for a given StackOverflow question. For example, we want a classifier that can
-predict posts about the Python language by tagging them `python`.
+The idea of the project is a simplified version of our
+[text classification tutorial](/doc/tutorials/deep). It explores the NLP problem
+of predicting tags for a given StackOverflow question. For example, we might
+want a classifier that can flag (or predict) posts about Python by tagging them
+with `python`.
```

Comment:
How about just "predict posts about Python by tagging them with `python`"? I'd rather avoid introducing the concept of flagging in this context (it's used for other purposes in our docs).

> In fact I'm going to add reviewing the usage of the term "flag" to #608. (Letting you know in case you want to address that one checkbox as part of this PR 🙂).
> ![image](https://user-images.githubusercontent.com/1477535/66505885-d8cd4700-ea91-11e9-8a0e-26489e9cc89d.png)

Reply:
That doesn't work well english wise - how about classify posts about python?

Reply:
Hm OK sure. I'm feeling apprehensive of the word "predict" here but I guess "classify" works. Or "predict which posts ... {something else}". Thanks!